group_project_v2/utils.py

```diff
@@ -350,7 +357,7 @@ def export_to_csv(data, target, headers=None):
 
 
 def named_tuple_with_docstring(type_name, field_names, docstring, verbose=False, rename=False):
-    named_tuple_type = namedtuple(type_name+"_", field_names, verbose=verbose, rename=rename)
+    named_tuple_type = namedtuple(type_name + "_", field_names, rename=rename)
```

Comment:
keeping the compatibility from py2 to py3 I guess I should remove verbose from the main function as well